### DISCUSSION TITLE
Lazer argon "pro" slider ball & follow points

### DISCUSSION THREAD

**Discussion by Ben6889** [ppy/osu#22722] [Ideas]
I am interested in seeing some other opinions on this. While play testing Argon "pro", I noticed that the slider ball while playing the slider, is very large and distracting especially when compared to an actual pro skin. Additionally, i feel that the follow points between notes could also be improved. For example: BTMC Freedom Dive skin has solid lines instead of several arrows as is present on argon pro. 

I would like to check with more people to see if maybe its just up to preference. Feel free to share your thoughts! 

Example of gameplay:

https://user-images.githubusercontent.com/123274555/220804846-90107016-1fc6-4f83-bee9-c402d82645da.mp4

**Comment by Ben6889** [comment]
I would also recommend more skin transparency!!!! (specifically w/ sliders and hit circles) due to it being harder to read notes below hit objects, i.e. Bursts.

**Comment by mk56-spn** [comment]
All of these changes with the possible exception of slider ball being distracted have been mentioned in previous argon suggestion threads
